apps/utils/locust_tests/high_volume_suite_post.py

```diff
@@ -0,0 +1,485 @@
+"""High Volume Load test suite for BFD Server endpoints."""
```

Comment:
Do we need to update one of the other files to call this test suite or is that going to be done later?

Reply:
I think we can do that later when we remove all `GET` suites. We did need to modify the infrastructure that runs the Regression Suite since it gates the deployment and we want to verify the `POST` endpoints as part of it whereas the High Volume Suite (HVS) is ran quite rarely and _usually_ only against ephemeral environments. We can probably hold off on modifying the HVS's supporting infrastructure to run the `POST`-variant until after the `GET` endpoints are removed.

Reply:
Sure, sounds good 👍